mod/dirfind.php

```diff
@@ -1,8 +1,10 @@
 <?php
 
 use Friendica\App;
+use Friendica\Core\Config;
 use Friendica\Core\System;
 use Friendica\Core\Worker;
+use Friendica\Network\Probe;
 
 require_once 'include/contact_widgets.php';
 require_once 'include/probe.php';
```

Comment:
Can you please check if this include is required after you replaced `probe_url()`?

Reply:
It appears it is not required, will remove.